data/json/recipes/armor/head.json

```diff
@@ -1829,6 +1829,12 @@
     "components": [ [ [ "link_sheet", 2 ] ], [ [ "chain_link", 50 ] ], [ [ "wire", 1 ] ], [ [ "rag", 4 ] ] ],
     "proficiencies": [ { "proficiency": "prof_chain_armour" } ]
   },
+  {
+    "result": "chainmail_hood",
+    "type": "recipe",
+    "copy-from": "chainmail_hood",
+    "components": [ [ [ "link_sheet", 2 ] ], [ [ "chain_link", 50 ] ], [ [ "wire", 1 ] ], [ [ "rag", 4 ] ] ]
+  },
```

Comment:
I'm a bit confused, what is this supposed to achieve? This is the same components as the recipe it's copied from. Can this be extra recipe be deleted?

Reply:
Yes, this is a duplicate that can go. I'm not at my desktop right now to fix it, but can get to it later.